packages/atomic/src/components/commerce/atomic-product-section-actions/atomic-product-section-actions.new.stories.tsx

```diff
@@ -0,0 +1,104 @@
+import type {Meta, StoryObj as Story} from '@storybook/web-components';
+import {wrapInCommerceInterface} from '@/storybook-utils/commerce/commerce-interface-wrapper';
+import {wrapInCommerceProductList} from '@/storybook-utils/commerce/commerce-product-list-wrapper';
+import {wrapInProductTemplate} from '@/storybook-utils/commerce/commerce-product-template-wrapper';
+import {parameters} from '@/storybook-utils/common/common-meta-parameters';
+import {renderComponent} from '@/storybook-utils/common/render-component';
+
+const {
+  decorator: commerceInterfaceDecorator,
+  play: initializeCommerceInterface,
+} = wrapInCommerceInterface({
+  skipFirstRequest: false,
+  type: 'product-listing',
+  engineConfig: {
+    context: {
+      view: {
+        url: 'https://sports.barca.group/browse/promotions',
+      },
+      language: 'en',
+      country: 'US',
+      currency: 'USD',
+    },
+  },
+});
+
+const {decorator: commerceProductListDecorator} = wrapInCommerceProductList();
+const {decorator: productTemplateDecorator} = wrapInProductTemplate();
+
+const meta: Meta = {
+  component: 'atomic-product-section-actions',
+  title: 'Atomic-Commerce/Product Template Components/ProductSectionActions',
+  id: 'atomic-product-section-actions',
+  render: renderComponent,
+  parameters,
+};
+
+export default meta;
+
+export const Default: Story = {
+  name: 'atomic-product-section-actions',
+  decorators: [
+    productTemplateDecorator,
+    commerceProductListDecorator,
+    commerceInterfaceDecorator,
+  ],
+  play: initializeCommerceInterface,
+  args: {
+    'slots-default': `<button class="btn btn-primary">Add to Cart</button>`,
+  },
+};
+
+export const WithMultipleActions: Story = {
+  name: 'With Multiple Actions',
+  decorators: [
+    productTemplateDecorator,
+    commerceProductListDecorator,
+    commerceInterfaceDecorator,
+  ],
+  play: initializeCommerceInterface,
+  args: {
+    'slots-default': `
+      <button class="btn btn-primary mr-2">Add to Cart</button>
+      <button class="btn btn-outline">Add to Wishlist</button>
+    `,
+  },
+};
+
+export const WithCustomButtons: Story = {
```

Comment:
This story seems a bit superfluous to me, but it's not a big deal.

Similar comment for other atomic-product-section-* .new.stories files